### DISCUSSION TITLE
Multilingual support (i18n)

### DISCUSSION THREAD

**Discussion by fmaida** [hugo-sid/hugo-blog-awesome#118] [Feature requests]
Would you accept a PR to add multilingual support through i18n to the theme?

**Comment by hugo-sid** [comment]
Sure ! I look forward to your PR.

**Comment by fmaida** [comment]
Thanks. This is an example of how the homepage should look if you set the defaultContentLanguage to Italian (it-it):

![Screenshot 2023-09-20 at 14-34-40 Home Vade Retro !](https://github.com/hugo-sid/hugo-blog-awesome/assets/605017/2cfdc18d-bc4d-4345-a80a-e45c9a9b1fad)


I'll make a PR as soon as I'll test that everything still works both in Italian and English.
